Dockerfile

```diff
@@ -5,7 +5,8 @@ WORKDIR /workspace
 
 # Make it runnable on a distroless image/without libc
 ENV CGO_ENABLED=0
-
+# Enviroment Variable used inside main to display current ARC version
+ENV ARC_VERSION=$VERSION
```

Comment:
Can we propagate this via go-build ldflags so that we can actually embed the value into the binary rather than reading it from a runtime environment variable? It's more error-prone as you can't accidentally alter the version number at run time, and slightly more maintainable as I think it's [a well-known practice](https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications) to embed the version number.

More concretely, we would be able to create a directory named `build/` in the repository root and create a source file `build/version.go`.

The source would look like:

```
package build

// This is overridden at build-time using go-build ldflags
var Version = "N/A"
```

In each Dockerfile, you add `-ldflags` to every `go build` invocation like:

```
go build -ldflags="-X 'github.com/actions-runner-controller/actions-runner-controller/build.Version=v0.26.0'"
```

Finally, `getVersion` functions would use `build.Version` instead of `os.Getenv`.

WDYT?

Reply:
That looks better! I will look into it and try to get a PR before the weekend . When is 0.26 expected merge ?